spec/apps/bootstrap-rejects/bootstrap-rejects.active-when.js

```diff
@@ -1,3 +0,0 @@
-export function activeWhen(location) {
```

Comment:
I deleted all the active-when files and created a `.spec` file instead that does what this file did, and more.